spec/system/admin/products_spec.rb

```diff
@@ -594,7 +594,7 @@
       click_button "Create"
 
       expect(page).to have_text "Attachment has an invalid content type"
-      expect(page).to have_text "Please upload the image in JPG, PNG, GIF, SVG or WEBP format."
+      expect(page).to have_text "Attachment is not a valid image"
```

Comment:
We need to take car of internationalization of this key. 
We should probably add the key `image_not_processable` here:

https://github.com/openfoodfoundation/openfoodnetwork/blob/fd74473bddcde873a2086f38b4e3443dc4c4d8c2/config/locales/en.yml#L89-L111

What do you think?

Reply:
thanks @jibees 
I'll work on this today and push the changes

Reply:
made above changes, resolving the conversation